importer/downloader.py

```diff
@@ -27,12 +49,14 @@ def main(argv=None):
             raise error
 
     for id in idlist:
-        downloadurl = "%s%s" % (fetchurl,id)
-        out_name = "Retractions/%s.xml" % (id) #make directory 'Retractions' first
-        print("Downloading article " + id)
-        with contextlib.closing(urllib.request.urlopen(downloadurl)) as response, open(out_name, 'wb') as out_file:
-            data = response.read()
-            out_file.write(data)
+        if(os.path.isfile("Retractions/" + id + ".xml") != True):
```

Comment:
`if (os.path.isfile(...))` should be sufficient here, no need to check against `True`